test/integration/072_experimental_parser_tests/test_all_experimental_parser.py

```diff
@@ -0,0 +1,35 @@
+from dbt.contracts.graph.manifest import Manifest
+import os
+from test.integration.base import DBTIntegrationTest, use_profile
+
+
+def get_manifest():
+    path = './target/partial_parse.msgpack'
+    if os.path.exists(path):
+        with open(path, 'rb') as fp:
+            manifest_mp = fp.read()
+        manifest: Manifest = Manifest.from_msgpack(manifest_mp)
+        return manifest
```

Comment:
This is neat! I know we were talking about doing more of this the other day, so it's great to see it in action

Reply:
I copy pasted this from @gshank 's partial parsing tests 🙃